code/game/objects/structures/crates_lockers/closets/secure/medical.dm

```diff
@@ -188,6 +188,8 @@
 	new /obj/item/device/defib_kit/compact/combat/adv/loaded/cbo(src)
 	new /obj/item/oddity/chem_book(src)
 	new /obj/item/storage/hcases/med(src)
+	if(prob(25))
```

Comment:
Idk why my review used the old file but, yknow. Change this thing.

Reply:
Might've been since you did it before the push.